README.md

```diff
@@ -5,44 +5,39 @@
 ```bash
 git clone -b linked-examples https://github.com/willcrichton/rust
 cd rust
-./x.py --stage 1 build
-export CUSTOM_RUSTDOC=$(pwd)/build/x86_64-apple-darwin/stage1/bin/rustdoc
+./x.py build
```

Comment:
Stage 1 has been the default since https://blog.rust-lang.org/inside-rust/2020/08/30/changes-to-x-py-defaults.html.